af.js

```diff
@@ -27,26 +31,32 @@ const af = {
     type: 'LAMBDA',
     arguments,
     statements
-  })
+  }),
+	exp: (exp, left, right) => ({
+		type: "EXP",
+		value: exp,
```

Comment:
Can the `value` ever be something besides an operator with the way you wrote https://github.com/Bradfield/languages-2017-01/pull/2/files#diff-fe30eed34176a05cc930283036bce555R39? If we don't have a valid value that isn't an operator, what do you think about calling this key `op` to avoid ambiguity?